Swift/BRCrypto/BRCryptoSystem.swift

```diff
@@ -34,6 +34,99 @@ public final class System {
     /// the networks, unsorted.
     public internal(set) var networks: [Network] = []
 
+    ///
+    /// Return the AddressSchemes support for `network`
+    ///
+    /// - Parameter network: the network
+    ///
+    /// - Returns: An array of AddressScheme
+    ///
+    public func supportedAddressSchemes (network: Network) -> [AddressScheme] {
+        switch network.currency.code {
```

Comment:
The Law of Demeter gently weeps...

Reply:
Hah, I feel younger now; remember past discussions, years ago.

Reply:
Maybe we have a chance to remove `Currency.codeAs{BTC,...}`?